jupyter.py

```diff
@@ -0,0 +1,8 @@
+"""Launch the root jupyter command
+
+Alias to jupyter_core
+"""
+
```

Comment:
Actually, that's a good point. We don't want people getting confused and giving us the version number of a trivial package that never gets upgraded instead of, say, `notebook.__version__`. Maybe it is best left out.